sdk/textanalytics/ai-text-analytics/src/textAnalyticsClient.ts

```diff
@@ -429,6 +437,20 @@ export class TextAnalyticsClient {
 
     this.client = new GeneratedClient(this.endpointUrl, internalPipelineOptions);
 
+    // let scopes;
+    // const env = this.isBrowser() ? (self as any).__env__ : process.env;
+
+    // switch (env["AZURE_AUTHORITY_HOST"]) {
+    //   case "https://login.microsoftonline.us":
+    //     scopes = "https://cognitiveservices.azure.us/.default";
+    //     break;
+    //   case "https://login.chinacloudapi.cn":
+    //     scopes = "https://cognitiveservices.azure.cn/.default";
+    //     break;
+    //   default:
+    //     scopes = DEFAULT_COGNITIVE_SCOPE;
+    // }
```

Comment:
Please note this is not a reliable way to detect the scope based on https://github.com/Azure/azure-sdk-for-js/issues/17563#issuecomment-921229288.

Reply:
@deyaaeldeen Do you mean that add the `audience` option into `TextAnalyticsClientOptions` to be used to detect the scope?